plugin_test.go

```diff
@@ -26,7 +27,6 @@ func TestPlugin(t *testing.T) {
 		g.It("Should upload files and directories", func() {
 			err := plugin.Exec()
 			g.Assert(err == nil).IsTrue(fmt.Sprintf("Failed to upload stuff: %s", err))
-			// g.Assert(uploaded == 2).IsTrue(fmt.Sprintf("Should have uploaded 2 files instead of %d files", uploaded))
```

Comment:
The hope would be tests continue to pass after modifying code ... did you forget to uncomment this test or is it still failing?

Reply:
When I forked the repo from your repo, this line was commented  - "// g.Assert(uploaded == 2).IsTrue(fmt.Sprintf("Should have uploaded 2 files instead of %d files", uploaded))"

Reply:
I retained the line as it is :)